README.md

```diff
@@ -1,54 +1,190 @@
 # Sudo Sandwich is an example Heroku Add-on
 
-Example Heroku add-on built with Rails for the Add-on Partner API v3.
+Example Heroku add-on built with Rails for the v3 of the Platform API for
```

Comment:
let's say "built with Rails" at the end of the sentence - that's less important than that it's for v3